content/10.diagnostics.md

```diff
@@ -168,7 +155,53 @@ Sensitivity and specificity metrics were obtained by comparison with results fro
+Current infection detection by LFTs enables the scale and speed of testing that is beneficial to managing viral spread.
+LFTs were made available freely to citizens in the United Kingdom at least until December 2021 [@DOI:10.1136/bmj.n1760] and to citizens of the United States in early 2022 [@url:https://www.whitehouse.gov/briefing-room/statements-releases/2022/01/14/fact-sheet-the-biden-administration-to-begin-distributing-at-home-rapid-covid-19-tests-to-americans-for-free].
+These tests are particularly useful for ruling out SARS-CoV-2 infection in cases where the likelihood of infection is low (e.g., asymptomatic individuals) and positives (including false positives) can be validated with testing by alternate means [@doi:10/hk34].
+
+##### Enzyme-Linked Immunosorbent Assay
+
+Enzyme-linked immunosorbent assay (ELISA) is a very sensitive immunoassay that can be considered a gold standard for the detection of biological targets, including antibodies and antigenic proteins [@pubmed:32310382].
+It can be used to generate either quantitative or qualitative results that can be returned within a few hours [@doi:10.1007/s00253-020-11061-5].
+ELISA builds on the idea that antibodies and antigens bind together to form complexes [@pubmed:32310382], utilizing enzymes that react with an antigen as reporter labels [@doi:10.1373/clinchem.2005.051532].
+The reagents are immobilized on a plate and anything in the sample that binds is retained [@url:https://www.bosterbio.com/protocol-and-troubleshooting/elisa-principle].
+The exact strategy for tagging with a reporter enzyme varies among different types of ELISA [@url:https://www.bosterbio.com/protocol-and-troubleshooting/elisa-principle; @pubmed:32310382].
+For COVID-19 diagnostics, ELISAs have been designed to detect all or part of the antigenic Spike protein [@doi:10.12688/wellcomeopenres.15927.1].
+
+One of these assays used two monoclonal antibodies specific to the nucleocapsid of SARS-CoV-2 to evalaute the relationship between the effect of (estimated) viral load on the ability of the assay to detect the SARS-CoV-2 antigen [@doi;10.1080/14787210.2021.1976144].
+This study analyzed naso-oropharyngeal 339 samples that were also analyzed with RT-qPCR, which here was used as a gold standard.
+RT-qPCR identified 147 samples as positive and 192 as negative.
+They estimated the overall sensitivity and specificity to be 61.9% and 99.0%, respectively.
+Sensitivity increased with higher C~t~.
+This study also assessed the performance of the ELISA test under different conditions in order to evaluate how robust it would be to the challenges of testing in real-world settings globally.
+Higher sensitivity was achieved for samples that were stored under ideal conditions (immediate placement in -80° C).
+Therefore, while immediate access to laboratory equipment is an advantage, it is not strictly necessary for ELISA to detect the antigen.
```

Comment:
I'm not sure its important, but it is cool that these ELISA can be used to detect if one has had a previous infection in the past or not, despite vaccination status, because they can detect nucleocapsid or envelope antigens in plasma. It might be useful for epidemiological and seroprevalence studies.